pkg/registry/core/service/strategy.go

```diff
@@ -162,6 +167,33 @@ func (serviceStatusStrategy) WarningsOnUpdate(ctx context.Context, obj, old runt
 	return nil
 }
 
+// GetAttrs returns labels and fields of a given object for filtering purposes.
+func GetAttrs(obj runtime.Object) (labels.Set, fields.Set, error) {
+	service, ok := obj.(*api.Service)
+	if !ok {
+		return nil, nil, fmt.Errorf("not a service")
+	}
+	return labels.Set(service.Labels), SelectableFields(service), nil
```

Comment:
```suggestion
	return service.Labels, SelectableFields(service), nil
```
Unnecessary type conversion.